packages/blocks/src/store/process-block-type.js

```diff
@@ -102,6 +124,9 @@ export const processBlockType =
 			),
 		};
 
+		// Stabilize any experimental supports before applying filters.
+		blockType.supports = stabilizeSupports( blockType.supports );
+
```

Comment:
Oh, good question. Unfortunately in this case I think we might need to apply it before the JS filters, because we have logic such as in [addAttributes](https://github.com/WordPress/gutenberg/blob/cc148c3a5b94f706aa152ee68075bc1c56e6ce79/packages/block-editor/src/hooks/font-family.js#L26) for font family that's registered against `blocks.registerBlockType` that will check against the stable name for the typography support.

I'll give this a bit more thought!

Reply:
I guess it could be done in both places. Hard to judge if it is worth the cost of doing so. Perhaps outreach could help get a feel for the possible impact of leaving it as only before the filters?